test/server.js

```diff
@@ -13,9 +13,61 @@ const config = {
     port: 5432,
     name: 'sensorthingstest',
     user: 'postgres',
-    pass: '12345678'
+    pass: '12345678',
+    hooks: {
+      'beforeValidate': (instance, options) => {
+        instance.properties = Object.assign({}, instance.properties, {
+          beforeValidate: 'beforeValidate'
+        });
+      },
+      'afterValidate': (instance, options) => {
+        instance.properties = Object.assign({}, instance.properties, {
+          afterValidate: 'afterValidate'
+        });
+      },
+      'beforeCreate': (instance, options) => {
+        instance.properties = Object.assign({}, instance.properties, {
+          beforeCreate: 'beforeCreate'
+        });
+      },
+      'afterCreate': (instance, options) => {
+        instance.properties = Object.assign({}, instance.properties, {
+          afterCreate: 'afterCreate'
+        });
+      },
+      'beforeDestroy': (instance, options) => {
+        instance.properties = Object.assign({}, instance.properties, {
+          beforeDestroy: 'beforeDestroy'
+        });
+      },
+      'afterDestroy': (instance, options) => {
+        instance.properties = Object.assign({}, instance.properties, {
+          afterDestroy: 'afterDestroy'
+        });
+      },
+      'beforeRestore': (instance, options) => {
+        instance.properties = Object.assign({}, instance.properties, {
+          beforeRestore: 'beforeRestore'
+        });
+      },
+      'afterRestore': (instance, options) => {
+        instance.properties = Object.assign({}, instance.properties, {
+          afterRestore: 'afterRestore'
+        });
+      },
+      'beforeUpdate': (instance, options) => {
+        instance.properties = Object.assign({}, instance.properties, {
+          beforeUpdate: 'beforeUpdate'
+        });
+      },
+      'afterUpdate': (instance, options) => {
+        instance.properties = Object.assign({}, instance.properties, {
+          afterUpdate: 'afterUpdate'
+        });
+      }
+    }
   }
```

Comment:
As I was creating a dedicated express app I realized the hook tests don't need one -- it is using sequelize to operate on the models directly. But event if it did, it seems to me the issue of enabling the hooks for the hooks tests but not for the other tests is related to the db module (db.js) essentially being a singleton; once it is instantiated and becomes READY, subsequent instantiations will return a promise the the originally instantiated instance. Since the majority of the tests will instantiate the db without the hooks, when the hooks test instantiates the db with a config that has the hooks, the db instance returned will be the one without the hooks because that instance was created first. I don't see a way around that.